integration-tests/test_client_options.py

```diff
@@ -17,3 +19,50 @@ def test_set_ansible_host_info(insights_client):
     ret = insights_client.run("--ansible-host=foo.example.com", check=False)
     assert "Could not update Ansible hostname" not in ret.stdout
     assert ret.returncode == 0
+
+
+def test_group(insights_client, tmp_path):
+    """Test --group option
+    This test generates an archive in offline mode and verifies if the group
+    specified in --group was created and packed
+    """
+
+    group_name = "testing-group"
+    archive_name = "archive.tar.gz"
+    archive_location = tmp_path / archive_name
+    tag_file = None
```

Comment:
since `tag_file` is a class that lives only until the tarfile is open, then I'd use `tag_file_content` instead to determine whether the tags were found

Reply:
done